packages/fxa-settings/package.json

```diff
@@ -60,6 +62,8 @@
     "eslint-plugin-react": "^7.19.0",
     "node-sass": "^4.14.0",
     "pm2": "^4.2.3",
-    "sass-lint": "^1.13.1"
+    "postcss-cli": "^7.1.1",
+    "sass-lint": "^1.13.1",
```

Comment:
_\*cough\*_ stylelint

Reply:
That's coming out here! https://github.com/mozilla/fxa/pull/5343